i18n/ja/code.json

```diff
@@ -38,15 +38,15 @@
   "header.community.stack_overflow": {
     "message": "スタックオーバーフロー"
   },
-  "footer.how_we_help": {
+  "footer.why_teradata_title": {
     "message": "ご支援できること"
   },
+  "footer.why_teradata": {
+    "message": "お役職別"
```

Comment:
Did this translation come from teradata.com?

Reply:
Yes. I just copied what they had in teradata.com.